lib/vbd_store.ml

```diff
@@ -21,7 +21,7 @@ module Make(Config : sig
     Lwt.catch
       (fun () -> Lwt_unix.mkdir vbd_list_dir 0o755)
       (function
-        | Unix.(Unix_error (EEXIST, "mkdir", dir)) when dir = vbd_list_dir -> Lwt.return_unit
```

Comment:
Is it just a compiler bug that it's moaning about this open?

Reply:
Maybe, I found this very puzzling. Maybe I should undo this change.

Reply:
Because the `EEXIST` variant is actually in the `Unix` module.

Reply:
This is the bug that's affecting xcp-idl too - I've had to do precisely this change there. Quite annoying actually.

Reply:
Don't despair... we will be able to move to `4.06.?` soon ;)

Reply:
So that means this PR is good to go then? :)

Reply:
I believe so